src/obb/server/controllers/index.cljc

```diff
@@ -0,0 +1,9 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.controllers.index
+  "Controller to handle the root/index request to the server")
+
+(defn handler
+  "Returns a intro to the game"
+  [request]
+  {:body "Hello World!"
+   :status 200})
```

Comment:
After looking to the routing definitions in `server/core`, I think an even better name would be `handlers`, or `http-handlers`